objectivec/docs/docs.readme.md

```diff
@@ -0,0 +1,78 @@
+# ONNX Runtime Objective-C API
+
+ONNX Runtime provides an Objective-C API.
+It can be used from Swift with a bridging header.
+
+The public headers are located [here](https://github.com/microsoft/onnxruntime/tree/master/objectivec/include).
+One can import them individually, or import the entire API with [onnxruntime.h](https://github.com/microsoft/onnxruntime/blob/master/objectivec/include/onnxruntime.h).
+
+## Example Usage in Objective-C++
+
+```objectivec
+#import <Foundation/Foundation.h>
+
+#import <onnxruntime.h>
+
+// Adds two numbers using ONNX Runtime.
+float add(float a, float b) {
+  // We will run a simple model which adds two floats.
+  // The inputs are named `A` and `B` and the output is named `C` (A + B = C).
+  // All inputs and outputs are float tensors with shape [1].
+  NSString* const kAddModelPath = @"/path/to/add.ort";
+
+  // ORT APIs take an optional NSError** parameter that will be set if an error occurs.
+  // Here, we will omit error handling (i.e., checking results and the NSError object) for brevity.
+  NSError* err = nil;
```

Comment:
Would be good to include a pointer on where they could see an example of the error handling. A link to a unit test is fine.

Or alternatively do the error check on a single call and then omit error handling in later calls for brevity.